ci-operator/step-registry/ipi/conf/azure/workers-marketimage/ipi-conf-azure-workers-marketimage-commands.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# TODO: move to image
+curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="/tmp/install-config-existingworkers-marketimage.yaml.patch"
+
+# az should already be there
+command -v az
+
+# set the parameters we'll need as env vars
+AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
+AZURE_AUTH_CLIENT_ID="$(<"${AZURE_AUTH_LOCATION}" jq -r .clientId)"
+AZURE_AUTH_CLIENT_SECRET="$(<"${AZURE_AUTH_LOCATION}" jq -r .clientSecret)"
+AZURE_AUTH_TENANT_ID="$(<"${AZURE_AUTH_LOCATION}" jq -r .tenantId)"
+
+# log in with az
+az login --service-principal -u "${AZURE_AUTH_CLIENT_ID}" -p "${AZURE_AUTH_CLIENT_SECRET}" --tenant "${AZURE_AUTH_TENANT_ID}" --output none
+# random select the market-place image
+images=$(az vm image list --all --offer ocp-worker --publisher redhat -o tsv --query "[?starts_with(urn,'RedHat:ocp-worker:ocp-worker')].urn")
+mapfile -t marketImages <<< "${images}"
+echo "choice market-place image from:" "${marketImages[@]}"
+imageTotal=${#marketImages[@]}
+echo "Image total: ${imageTotal} "
+selected_image_idx=$((RANDOM % ${imageTotal}))
+image=${marketImages[selected_image_idx]}
+echo "$(date -u --rfc-3339=seconds) - Selected image ${image}"
+IFS=':' read -ra imageInfo <<< "${image}"
+echo "total:" ${#imageInfo[@]} "${imageInfo[@]}"
```

Comment:
I think this is debug, if no need, we can remove it, or only pint out `"${imageInfo[@]}"`